### PR TITLE
fix: fix array generic type should start with [

### PR DESCRIPTION
**Pull request by killagu**

### CODE REVIEW

**Comment by killagu** [comment]
cc: @mansonchor

File: lib/utils.js [hunk 1]
```diff
@@ -7,7 +7,11 @@ function normalizeGeneric(type) {
   if (!type.generic) return '';
   let str = '';
   for (const item of type.generic) {
-    str += ('#' + item.type + normalizeGeneric(item));
+    str += '#';
+    if (item.isArray) {
```
Comment (mansonchor) [inline lib/utils.js:11]: 递归泛型的也要处理吧

Comment (mansonchor) [reply lib/utils.js:11]: 忽略我...这里是递归调用

**Review by gxcsoccer — APPROVED** [review]
+1

**Comment by mansonchor** [comment]
+1

**Comment by fengmk2** [comment]
+1

**Comment by killagu** [comment]
`+ sofa-hessian-node@1.7.1`

**Pull request merged**